.github/workflows/buildsCI.yaml

```diff
@@ -302,8 +302,7 @@ jobs:
       continue-on-error: true 
       run: |
         export LRS_LOG_LEVEL="DEBUG";
-        cd build/${{env.LRS_RUN_CONFIG}}/
-        ./live-test -d yes -i [software-device]        
+        cd build/${{env.LRS_RUN_CONFIG}}/  
```

Comment:
@noacoohen, do we need `U20_U24_ST_Py_EX_CfU_LiveTest` section, if no live tests are ran?

Reply:
BTW the name is wrong right? Its U22 U24?

Reply:
@Arun-Prasad-V ,This is the only build that compiles Ubuntu with Examples & Tools or with Check for Updates so I didn't remove it

Reply:
@Nir-Az, yes, I'll change it

Reply:
@noacoohen, okay got it. Then, does it make sense to remove `_LiveTest` from the name and also remove the `Test` section if we are not going to test anything?

Reply:
Done